src/client.ts

```diff
@@ -102,7 +102,7 @@ class Client extends EventEmitter implements IClient {
         return definitions.map((d) => {
             const enabled = this.unleash.isEnabled(d.name, context);
             const variant = enabled
-                ? this.unleash.forceGetVariant(d.name, context)
+                ? this.unleash.getVariant(d.name, context)
```

Comment:
I tested the original issue: https://github.com/Unleash/unleash-client-node/pull/307 and getVariant seems to be working fine now. getVariant is necessary since forceGetVariant was bypassing the strategy variant resolution

Reply:
Run the original script and % distribution was correct

Reply:
That's good! If possible, I'd like to have @sighphyre's input on this before merging it, though.

Reply:
Yeah, we've chatted about it, the original script does now show the correct distributions without this. That patch is over a year old and we've made a lot of changes, so it looks it's no longer relevant